draft-ietf-quic-transport.md

```diff
@@ -4313,6 +4531,15 @@ the range from 0xFE00 to 0xFFFF.
 
 Issue and pull request numbers are listed with a leading octothorp.
 
+## Since draft-ietf-quic-transport-12
+
+- Enable server to transition connections to a preferred address (#560,#1251).
+- No more stream 0.
```

Comment:
Stream 0 still exists.  Maybe "Stream 0 is no longer reserved for the cryptographic handshake.  Instead, CRYPTO_HS frames are used."